### DISCUSSION TITLE
Log in from Microsoft work account

### DISCUSSION THREAD

**Discussion by netzer-git** [community/community#47952] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi!

After pairing my personal account with my Microsoft work account, I'm still getting a message about my free-60-days are over and I need to pay, isn't copilot supposed to be free to use for Microsoft workers?

Thanks!

**Comment by OwaisAzad** [comment]
As far as I know, GitHub Copilot is a paid service and does not come free with a Microsoft work account. However, if you are a member of the GitHub Student Developer Pack, you can use Copilot for free for a limited time. Additionally, some GitHub Enterprise plans include access to Copilot for enterprise users.

If you are using Copilot as an individual and have already used up your free trial period, you will need to purchase a subscription to continue using the service.

**Comment by Akash1134** [comment, marked as answer]
Hi @netzer-git As per your account details, you have access to Copilot for Individuals via your Microsoft account. Additionally, you've got Copilot access for using your personal GitHub account associated with the employer. Have you checked your work email to associate your personal account? Thanks!

**Reply by km-Poonacha** [reply]
Did everyone at Microsoft receive access to GitHub Copilot when it was released ?
